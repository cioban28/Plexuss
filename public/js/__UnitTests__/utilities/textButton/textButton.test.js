import React from 'react';
import TextButton from './../../../bundles/utilities/textButton/textButton';
import renderer from 'react-test-renderer';

describe('TextButton', () => {

  it('button has active class when active prop passed', () => {
      const component = renderer.create(
        <TextButton title="Facebook" active="Facebook"></TextButton>
      );

      let tree = component.toJSON();
      expect(tree).toMatchSnapshot();
  });



  it('button not active', () => {
      const component = renderer.create(
        <TextButton title="Facebook" active="Google"></TextButton>
      );

      let tree = component.toJSON();
      expect(tree).toMatchSnapshot();
  });

  
});

