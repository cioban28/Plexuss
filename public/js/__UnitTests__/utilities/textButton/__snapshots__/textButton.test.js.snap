// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextButton button has active class when active prop passed 1`] = `
<div
  className="active"
  onClick={undefined}
>
  Facebook
</div>
`;

exports[`TextButton button not active 1`] = `
<div
  className=""
  onClick={undefined}
>
  Facebook
</div>
`;
